src/groups/FBX.tsx

```diff
@@ -0,0 +1,245 @@
+// FBX.tsx
+import * as React from "react";
+import * as THREE from "three";
+import { FBXLoader } from "three/examples/jsm/loaders/FBXLoader";
+import PropTypes from "prop-types";
+import exact from "prop-types-exact";
+import { useViewContext, useAnimationFrame } from "../utils/hooks";
+import Group, { GroupProps, GroupPropTypes } from "./Group";
+import { Label } from "../primitives";
+
+const { useEffect, useMemo, useRef, memo } = React;
```

Comment:
You can combine them on top like so.

`import React, { useEffect, useMemo, useRef, memo } from "react";`

Reply:
So, for some reason react-three-fiber needs `import * as React from "react";`
which doesn't work with the comma and destructuring. 
I tried a few times and reverted back to this.

Reply:
Ah its probably because of typescript...

Reply:
Ah nevermind, it is a linter error. My bad, `THREE` is the one that needs to be imported with `* as THREE`